lib/mayaUsd/fileio/utils/meshWriteUtils.cpp

```diff
@@ -828,17 +828,9 @@ UsdMayaMeshWriteUtils::writeUVSetsAsVec2fPrimvars(const MFnMesh& meshFn,
             continue;
         }
 
-        // XXX: We should be able to configure the UV map name that triggers this
-        // behavior, and the name to which it exports.
-        // The UV Set "map1" is renamed st. This is a Pixar/USD convention.
-        TfToken setName(uvSetNames[i].asChar());
-        if (setName == UsdMayaMeshPrimvarTokens->DefaultMayaTexcoordName.GetText()) {
-            setName = UsdUtilsGetPrimaryUVSetName();
```

Comment:
The removes the automatic "map1"-> "st" renaming on export. Should I create an env var for shops that have internal tools that depend on this behavior?

Reply:
Yes, definitely. Having the Maya USD export portion of our asset builds suddenly writing `map1` primvars instead of `st` primvars would be a non-trivial pipeline change for us, so at least for now, we'll need to be able to opt out of that.

Including a setting for import, maybe a pair of `TfEnvSettings` like this?:
```C++
TF_DEFINE_ENV_SETTING(
    MAYAUSD_IMPORT_PRIMARY_UV_SET_AS_MAP1,
    false,
    "Translates the primary UV set in USD to the map1 UV set in Maya. "
    "When disabled, UV set names are translated directly (st in USD becomes st in Maya).");

TF_DEFINE_ENV_SETTING(
    MAYAUSD_EXPORT_MAP1_AS_PRIMARY_UV_SET,
    false,
    "Translates the map1 UV set in Maya to the primary UV set in USD. "
    "When disabled, UV set names are translated directly (map1 in Maya becomes map1 in USD).");
```

Thanks! :)

Reply:
Yes agreed, having the env variable to make a switch would be useful. Thanks